processor/sumologicprocessor/cloud_namespace_processor.go

```diff
@@ -0,0 +1,79 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package sumologicprocessor // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/sumologicprocessor"
+
+import (
+	conventions "go.opentelemetry.io/collector/model/semconv/v1.6.1"
```

Comment:
I'm sorry, I didn't think to check for this yesterday: can this be easily replaced with `go.opentelemetry.io/collector/semconv`? Right now the processor uses `go.opentelemetry.io/collector/model` which doesn't receive releases anymore and has some old dependencies on the Go net and Google gRPC modules. There were just some CVEs in these modules and I want to make sure we keep the dependency tree clean.